### PR TITLE
add `withDrawAll` function

### LINKED ISSUE #31: [NFT] - Withdraw function

### PR DESCRIPTION
**Pull request by robertu7**
Base branch: `feature/mint-script`

### CODE REVIEW

**Review by devformatters — APPROVED** [review]
Go. 👨🏿‍🏫

File: contracts/BatchNFT.sol [hunk 1]
```diff
@@ -27,6 +27,16 @@ contract BatchNFT is ERC721, Ownable {
         setSupply(supply_);
     }
 
+    receive() external payable {}
```
Comment (guoliu) [inline contracts/BatchNFT.sol:30]: What is the point of this function?

**Review by guoliu — APPROVED** [review]
🪴

File: contracts/BatchNFT.sol (hunk 1 above)
Comment (robertu7) [reply contracts/BatchNFT.sol:30]: just make the contract can receive Ether, now only for [testing](https://github.com/thematters/matters-avatar/pull/34/files#diff-ee4c354a2a46ddb89a4678af8d6e4749fefefd658910e922fe70c2ff2f965b64R75)

**Pull request merged**